README.md

```diff
@@ -2,4 +2,6 @@
 [![codecov](https://codecov.io/gh/translationCoreApps/translationNotes/branch/master/graph/badge.svg)](https://codecov.io/gh/translationCoreApps/translationNotes)
 
 # translationNotes
```

Comment:
The definition on the left seems like the legitimate description for tN and the one on the right sounds more like tW